MockKitUI/Assets.xcassets/iCGM.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "iCGM.pdf",
```

Comment:
[nit] Perhaps `mockCGM`?

Reply:
these were the names of the files as given to me by Paul.

Reply:
I agree mockCGM is better; these are not "iCGM".

Reply:
The`imageset` is named `mockCGM`.  The name of the file is just the one from design.